src/api/sql.js

```diff
@@ -3,6 +3,10 @@ var $ = require('jquery');
 var Mustache = require('mustache');
 var Promise = require('./promise');
 
+var errorMessages = {
```

Comment:
I did it this way to have all error messages (so far we only have one) in an object instead of having a bunch of error messages each in one variable.

Changing it.